openpype/hosts/maya/plugins/publish/validate_skeleton_rig_content.py

```diff
@@ -0,0 +1,101 @@
+import pyblish.api
+from maya import cmds
+
+from openpype.pipeline.publish import (
+    PublishValidationError,
+    ValidateContentsOrder
+)
+
+
+class ValidateSkeletonRigContents(pyblish.api.InstancePlugin):
+    """Ensure skeleton rigs contains pipeline-critical content
+
+    The rigs optionally contain at least two object sets:
+        "skeletonMesh_SET" - Set of the skinned meshes
+                             with bone hierarchies
+
+    """
+
+    order = ValidateContentsOrder
+    label = "Skeleton Rig Contents"
+    hosts = ["maya"]
+    families = ["rig.fbx"]
+
+    accepted_output = ["mesh", "transform", "locator"]
```

Comment:
This would be best as a `set` since we're doing lookups into it.
```suggestion
    accepted_output = {"mesh", "transform", "locator"}
```